requirements.txt

```diff
@@ -0,0 +1,8 @@
+codacy-coverage==1.3.11
```

Comment:
We have too many requirements in single file `requirements.txt` and we've locked all of these. We should split these requirements and determine which versions we must lock and which not.

Let's see which type of requirements we could have:
1. The project's requirements - the same in [setup.py](https://github.com/devopshq/crosspm/blob/master/setup.py#L84-L91).
2. Test's requirements - [pytest and others](https://github.com/devopshq/crosspm/blob/master/setup.py#L79-L81)
3. CI requirements - coverage, codace.

We should lock 2 and 3 types of requirements and we **should not** fix versions of the first type of requirements. 

Why? Because we really don't know which version has been installed in end user's environment

I advice to split `requirements.txt` to three files: 
- requirements.txt (without `==`)
- requirements-test.txt (with `==`)
- requirements-ci.txt (with `==`)

The last contains it:
```
-r requirements.txt
-r requirements-test.txt
coverage==4.5
codacy-coverage==1.3.11
```

And use it in `.travis-ci.yml` like this `pip install -r requirements-ci.txt`

Reply:
Thank you for your advices! That's why I like code review as a concept - this is the fastest way to gain experience in programing (or somewhere else) based on the experience of the rest of the people.
I will patch code based on your remarks in a nearest future.